bin/vue-init

```diff
@@ -158,3 +158,7 @@ function downloadAndGenerate (template) {
     })
   })
 }
+
+function isLocal (str) {
```

Comment:
`isLocal` implies more than just that str is string and not an url. In my opinion `exist` should also be called here.
+ you can easily add test for it ;)